src/Homework6/reservation_apartment.py

```diff
@@ -0,0 +1,112 @@
+"""Создайте  модель из жизни. Это может быть бронирование комнаты
+ в отеле, покупка билета в транспортной компании, или простая РПГ.
+  Создайте несколько объектов классов, которые описывают ситуацию
+   Объекты должны содержать как атрибуты так и методы класса для
+    симуляции различных действий. Программа должна инстанцировать
+     объекты и эмулировать какую-либо ситуацию - вызывать методы,
+      взаимодействие объектов и т.д.
+"""
+
+import datetime
+from datetime import date
+
+
+class Apartment(object):
+    def __init__(self, adress):
+        self.adress = adress
+        self.print_welcome(guest.get_first_name())
+        self.clean(reservation.get_checkin_date(),
+                   reservation.get_checkout_date())
+        self.set_agent('Natali')
+
+    def set_agent(self, agent):
+        self.agent = agent
+        print(f"{agent} will be on call for Apartment {self.adress}")
+
+    def clean(self, checkin_date, checkout_date):
+        print(f"Apartment {self.adress} will be cleaned on"
+              f" {checkin_date} and {checkout_date}")
+
+    def print_welcome(self, first_name):
+        print(f"Welcome at the Apartment {self.adress}, "
+              f"{first_name} good stay!")
+
+
+class Guest(object):
+    def __init__(self, first_name, last_name, num_adults, num_children):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.num_adults = num_adults
+        self.num_children = num_children
+
+    def get_first_name(self):
+        return self.first_name
+
+    def __repr__(self):
```

Comment:
why repr?

Reply:
When i use firma.print_lists() and __str__, i see:
The reservation list is: [<__main__.Reservation object at 0x0045DE50>, <__main__.Reservation object at 0x004A46B0>]
else:
The reservation list is: [Reservation: (2020-03-08 ~ 2020-03-11, Kolasa), Reservation: (2020-03-08 ~ 2020-03-13, Geroev)]

Reply:
yes but you can create \_\_str__ method instead and it would be used by str correctly